<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Network</class>
 <widget class="QWidget" name="Network">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="ipLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>60</y>
     <width>119</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Server IP</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="serverIp">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>60</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QTextEdit" name="serverPort">
   <property name="geometry">
    <rect>
     <x>160</x>
     <y>110</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="portLabel">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>110</y>
     <width>119</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Server port</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="isServer">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>10</y>
     <width>91</width>
     <height>22</height>
    </rect>
   </property>
   <property name="text">
    <string>Server</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
